package.json

```diff
@@ -19,6 +19,7 @@
     "@snapshot-labs/snapshot-metrics": "^1.0.0",
     "@snapshot-labs/snapshot-sentry": "^1.1.0",
     "@snapshot-labs/snapshot.js": "^0.3.68",
+    "@xmtp/bot-starter": "^1.1.4",
```

Comment:
Got this error while doing a `yarn install`

```
error @xmtp/xmtp-js@10.1.1: The engine "node" is incompatible with this module. Expected version ">=18". Got "16.20.0"```

Reminding that all our infra run on node 16, because 18 is not supported

Reply:
Yes this require Node v18

Reply:
Will probably move `snapshot-webhook` to Heroku for now to support v18